### ISSUE TITLE
About the fetch url you are using

### ISSUE THREAD

**Issue by JerryLi620** [JasonWYZE/Rate_My_Professor_Extension#2]
Hi Jason, I am also developing a rate my professor extension for my school. Since RMP does not have an api, I searched around and found most developers used "https://search-production.ratemyprofessors.com/*" as the fetch url. However, when I try to use it , it seems like the website is not working. I am wondering if this url is still working.

**Comment by JasonWYZE** [comment]
Hi,

This URL is still working. I can share my code with you later if you still
need help.

JerryLi620 ***@***.***>于2022年12月19日 周一04:45写道：

**Comment by JerryLi620** [comment]
I get an Error 503 Backend unavailable, connection timeout error when I try to fetch the url. Did I miss any important step? Thanks for your help!

**Comment by JasonWYZE** [comment]
Did you add "host_permissions": ["
https://search-production.ratemyprofessors.com/*" ] on your manifest.json
file？

JerryLi620 ***@***.***> 于2022年12月28日周三 08:42写道：

**Comment by JerryLi620** [comment]
Yes. I am wondering what appears if you clicks the url. For example, here is a sample url I generated: https://search-production.ratemyprofessors.com/solr/rmp/select/?solrformat=true&rows=2&wt=json&q=sarafova+s+AND+schoolname_t:Davidson+College. When I clicked it, it shows me the following website, and the fetch will give me the html of this website instead of a json file. 
<img width="704" alt="截屏2022-12-30 12 05 44" src="https://user-images.githubusercontent.com/90734286/210033440-f469bd9a-7b7a-4581-a17b-9d5181953b5b.png">
Thanks for your time. I really appreciate it!

**Comment by JerryLi620** [comment]
previously I thought it was an vpn error, but I let my friends in US to test it. They also got an 503 error which is really annoying.

**Comment by JasonWYZE** [comment]
I just tested it on my side and it is not working now. Sorry about that. I
am also curious about what is going on. However, I was planning to write my
own fetch API since the original one doesn't work well. Let me know what
you think.

JerryLi620 ***@***.***> 于2022年12月29日周四 23:11写道：

**Comment by JerryLi620** [comment]
That will be amazing! I think RMP is using Graphql to store the data right now, and I am not familiar with it. Please let me know if you finish writing the API or if I can help with anything. Thanks for your help!
